packages/standard-minifier-js/plugin/minify-js.js

```diff
@@ -1,161 +1,105 @@
 import { extractModuleSizesTree } from "./stats.js";
 
 Plugin.registerMinifier({
-  extensions: ['js'],
-  archMatching: 'web'
-}, function () {
-  var minifier = new MeteorBabelMinifier();
-  return minifier;
-});
-
-function MeteorBabelMinifier () {};
-
-MeteorBabelMinifier.prototype.processFilesForBundle = function(files, options) {
-  var mode = options.minifyMode;
-
-  // don't minify anything for development
-  if (mode === 'development') {
-    files.forEach(function (file) {
-      file.addJavaScript({
-        data: file.getContentsAsBuffer(),
-        sourceMap: file.getSourceMap(),
-        path: file.getPathInBundle(),
+    extensions: ['js'],
+    archMatching: 'web',
+  }, 
+  () => new MeteorBabelMinifier()
+);
+
+class MeteorBabelMinifier {
```

Comment:
This class could be renamed since it no longer uses Babel.

Reply:
Fixed this.